packages/server/graphql/types/Threadable.ts

```diff
@@ -74,7 +74,7 @@ const Threadable = new GraphQLInterfaceType({
       return Task
     }
 
-    if ('options' in threadable) {
+    if ('title' in threadable) {
```

Comment:
If we change the DB queries to include the poll options in a single DB hit, then it'd work. Otherwise, there are no `options` when fetching the `poll`